src/loaders/ObjectLoader.js

```diff
@@ -466,8 +466,21 @@ THREE.ObjectLoader.prototype = {
 
 				case 'PerspectiveCamera':
 
-					object = Object.assign(
-							new THREE.PerspectiveCamera(), data );
+					object = new THREE.PerspectiveCamera(
+							data.fov, data.aspect, data.near, data.far );
+
+					if ( data.focus !== undefined ) object.focus = data.focus;
+					if ( data.zoom !== undefined ) object.zoom = data.zoom;
+
+					if ( data.filmGauge !== undefined ) {
+
+						if ( data.view !== null )
+							object.view = Object.assign( {}, data.view );
```

Comment:
Okay.  I guess I'd decouple the reading of the parameters from their time of implementation in the ThreeJS scene JavaScript exporter.  This is so minor, but be aware that this is likely reading from exporters (Blender, fbx converter, etc.) that are often partially implemented (no guarantee which variables are written out or whether they have anywhere consistent support with the reader) and those barely working (it is always like this unfortunately) exporters are more likely to be the ones writing the file than the ThreeJS scene JavaScript exporter.